### ISSUE TITLE
Error fetching request ... from database - Conversion error

### ISSUE THREAD

**Issue by vitorquintanilha** [LucasPickering/slumber#400]
**Did you [search](https://github.com/LucasPickering/slumber/issues) for existing issues already?**
:heavy_check_mark: 

**Describe the bug**
When I start or refresh slumber I see the following error:
```
╔Error═══════════════════════════════════════════════════════════════╗
║Error fetching request [profile=local; recipe=soundcheck_graphql_0] ║
║from database                                                       ║
║└┬Conversion error from type Blob at index: 8, parse error at line  ║
║2, column 1  |2 | cf-access-token:  | ^                             ║
║ └─parse error at line 2, column 1  |2 | cf-access-token:  | ^      ║
║                                                                    ║
║                                                                    ║
║                                                                    ║
║                                                                    ║
║                                                                    ║
╚════════════════════════════════════════════════════════════════════╝
```

**To Reproduce**
_Steps to reproduce the behavior_

Use this config:
```
profiles:
  production:
    data:
      host: https://some.cool.url/api
  local:
    data:
      host: http://localhost:7007/api

chains:
  token:
    source: !command
      command: [bash, -c, exit 1]

requests:
  graphql_0: !request
    headers:
      cf-access-token: '{{chains.token}}'
    method: POST
    url: "{{host}}/graphql"
    body: !json
      query: |
        { types }
```

2. Run a bunch of successful requests using the TUI
3. Refresh (F5)

**Expected behavior**
Clean refresh

**Screenshots**
![image](https://github.com/user-attachments/assets/8d86adca-abca-46a1-8405-a7c2b6949986)

**Version (please complete the following information):**

- OS: Linux ... 6.6.56-1-lts x86_64 GNU/Linux
- Terminal: alacrity + tmux
- Slumber Version: 2.1.0

**Additional context**
The value for the `cf-access-token` is a very long string (~800 chars)

_____ EDIT  NOTES _______

I actually found out this happens when the `!command` fails, it's `stdout` is empty, and the header is set to an empty string. I've updated it accordingly.

**Comment by LucasPickering** [comment]
Thanks for the report, I just merged a fix. I'll put out a new version this week.

**Comment by vitorquintanilha** [comment]
Thank you @LucasPickering  for `slumber`, I enjoy it a lot :)

**Comment by LucasPickering** [comment]
Fix for this is out in 2.2.0
